src/mlpack/tests/split_data_test.cpp

```diff
@@ -94,20 +94,35 @@ void CheckDuplication(const Row<size_t>& trainLabels,
     BOOST_REQUIRE_EQUAL(counts[i], 1);
 }
 
-BOOST_AUTO_TEST_CASE(SplitDataResultMat)
+BOOST_AUTO_TEST_CASE(SplitShuffleDataResultMat)
 {
   mat input(2, 10);
-  size_t count = 0; // count for putting unique sequential values
+  size_t count = 0; // counter for unique sequential values
```

Comment:
```suggestion
  size_t count = 0; // Counter for unique sequential values.
```

To be consistent with the rest of the codebase.